inc/notifiableinterface.class.php

```diff
@@ -87,4 +87,11 @@ public function refreshPersistedNotifications();
     * @return boolean True if message may be sent
     */
    public function isNotifiable();
+
+   /**
+    * Find agents to be notified
+    * @param integer $notifiableId
+    * @return array
+    */
+   public function findNotifiableAgents($notifiableId);
```

Comment:
GetNotifiableAgents() is a better name (because it tells the method returns something). This also contributes to similar method naming convention with Formcreator plugin (I introduced several methods having the same role for a similar need)